### PR TITLE
Allow min_update_period to be overridden

### PR DESCRIPTION
**Pull request by ajoubertza**
This allows the default minimum update period to be overridden via a property. It can be set when launching the device from the command line.

Not sure how to make a test for this, so here's an example of the reduced update period for the katsim dish simulator.   Note that calls to [pre_update](https://github.com/ska-sa/katsim/blob/01c19ba9b60f98bcbc01915d24635c9526173731/katsim/ska_mpi_dsh_lmc_override.py#L511) depend on client access to the device, so it is not regular.  However, we can see that the times are now close to 0.2 seconds instead of the default 0.99 seconds.  Scroll all the way to right to see the new argument:
`--put-device-property mid_dish_0026/elt/master:min_update_period:0.2`
```
$ kat-tango-sim-launcher.py --name mid_dish_0026/elt/master --class DishElementMaster --name mid_dish_0026sim/elt/master --class DishElementMasterSimControl --server-command DishElementMaster --server-instance mid_dish_0026 --port 4052 --file-name katconfig/static/antennas/s0026_sim.conf --put-device-property mid_dish_0026sim/elt/master:model_key:mid_dish_0026/elt/master --put-device-property mid_dish_0026/elt/master:min_update_period:0.2
Setting device 'mid_dish_0026/elt/master' property 'min_update_period': '0.2'
Setting device 'mid_dish_0026sim/elt/master' property 'model_key': 'mid_dish_0026/elt/master'
Starting TANGO device server:
'DishElementMaster' 'mid_dish_0026' '-ORBendPoint' 'giop:tcp::4052' '-file=/tmp/tmpzk3FX3'
Can't create notifd event supplier. Notifd event not available
Ready to accept request
pre_update called - dt 0.861226081848
pre_update called - dt 0.399704933167
...
pre_update called - dt 0.200511932373
pre_update called - dt 0.800007104874
...
pre_update called - dt 0.202538013458
pre_update called - dt 0.207702875137
pre_update called - dt 0.799002170563
```

JIRA (SKA):  [SAR-155](https://jira.skatelescope.org/browse/SAR-155)

### CODE REVIEW

**Review by samueltwum1 — APPROVED** [review]
LGTM

**Review by SKAJohanVenter — APPROVED** [review]

File: doc/intro.rst [hunk 1]
```diff
@@ -130,10 +130,25 @@ in turn start up the device server process, all in one go.
     $ tango-simlib-launcher --name mkat_sim/weather/1 --class Weather\
                             --name mkat_simcontrol/weather/1\
                             --class WeatherSimControl\
-                            --server-command weather-DS --port 0\
+                            --server-command ./weather-DS --port 0\
```
Comment (samueltwum1) [inline doc/intro.rst:133]: I forgot to ask this: passing `weather-DS` instead of `./weather-DS` should still work. So why the change?

Comment (ajoubertza) [reply doc/intro.rst:133]: When I was testing out this command, it failed because `weather-DS` wasn't on the path.  Adding the `./` just helps the example work a little more easily.

Comment (samueltwum1) [reply doc/intro.rst:133]: okay. thanks
